x/ibc/07-tendermint/update.go

```diff
@@ -82,30 +85,17 @@ func checkValidity(
 			header.TrustedValidators, consState.NextValidatorsHash, tvalHash,
 		)
 	}
-	// assert trusting period has not yet passed
-	if currentTimestamp.Sub(consState.Timestamp) >= clientState.TrustingPeriod {
```

Comment:
Checked here: https://github.com/tendermint/tendermint/blob/v0.33.6/lite2/verifier.go#L214

Reply:
could you note that all these checks are done by tm in the code?